perun-base/src/test/resources/test-roles.yml

```
@@ -205,5 +205,14 @@ perun_policies:
       - PROXY:
     include_policies: []
 
+  test_filter-getMembersPage_policy:
+    policy_roles:
+      - GROUPADMIN: Group
+      - GROUPOBSERVER: Group
+      - GROUPMEMBERSHIPMANAGER: Group
+      - PERUNOBSERVER: Group
```

The PerunObserver role is not related to any object.

```suggestion
      - PERUNOBSERVER: 
```